apps/stripe/vitest.config.ts

```diff
@@ -25,6 +26,24 @@ export default defineConfig({
           setupFiles: "./src/__tests__/setup.integration.ts",
         },
       },
+      {
+        extends: true,
+        test: {
+          globalSetup: "./src/__tests__/integration/dynamodb/global-setup.integration-dynamo.ts",
+          include: ["src/__tests__/integration/dynamodb/**/*.test.{ts,ts}"],
+          name: "integration:dynamodb",
+          setupFiles: "./src/__tests__/integration/dynamodb/setup.integration-dynamo.ts",
+          poolOptions: {
+            forks: {
+              /*
+               * todo this likely have side effect with test isolation, so maybe we need different way to force sequence
```

Comment:
I think we can override sequence here from main config and set it up to be sequential instead of random